<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Vault</class>
 <widget class="QWizardPage" name="Vault">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>501</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbUnlockFile">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>460</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Unlock</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>464</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lpathOfFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Select file from the dropdown list to unlock.</string>
   </property>
  </widget>
  <widget class="QLabel" name="lSelectFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>26</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Files locked by you:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbLockFile">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>87</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Click to lock file</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
